module/VuFindSearch/src/VuFindSearch/Backend/Solr/QueryBuilder.php

```diff
@@ -210,6 +212,64 @@ public function build(AbstractQuery $query, ?ParamBag $params = null)
         return $newParams;
     }
 
+    /**
+     * Converts a simple query (Query) into an advanced one (QueryGroup) if part of it should be an exact query.
+     * This only supports a single exact query (surrounded with quotes) combined with a non-exact query.
+     * Logical operators can be used, but not parenthesis or field names.
```

Comment:
Nitpick: I'd use the plural here:

```suggestion
     * Logical operators can be used, but not parentheses or field names.
```